src/components/ExpandedViewMode/ExpandedViewModeContent/ActivityHistory/ActivityHistory.css

```diff
@@ -0,0 +1,25 @@
+.history {
+  padding: 10px;
+  margin: 20px 30px;
+  color: rgb(0, 0, 0);
+}
+
+.history {
+  display: flex;
+}
+.historyPic {
```

Comment:
for consistency with the rest of the code, can you use word-word instead of wordWord, in classnames?
 e.g. 
.history-pic
instead of
.historyPic

Reply:
ok, i'm working on it